deployment/openshift/web_dc.yaml

```diff
@@ -50,7 +50,9 @@ objects:
             "REACT_APP_KEYCLOAK_URL": "${REACT_APP_KEYCLOAK_URL}",
             "REACT_APP_APPLICATION_NAME": "${REACT_APP_APPLICATION_NAME}",
             "REACT_APP_WEB_BASE_CUSTOM_URL": "${REACT_APP_WEB_BASE_CUSTOM_URL}",
-            "REACT_APP_USER_ACCESS_PERMISSIONS": ${REACT_APP_USER_ACCESS_PERMISSIONS}
+            "REACT_APP_USER_ACCESS_PERMISSIONS": ${REACT_APP_USER_ACCESS_PERMISSIONS},
+            "REACT_APP_FORMIO_JWT_SECRET": "${REACT_APP_FORMIO_JWT_SECRET}",
+            "DESIGNER_ROLE_ID": "61e557ebe8a170e1b2bb565a
```

Comment:
The `DESIGNER_ROLE_ID` would not be needed here. The app accesses this one using `REACT_APP_STAFF_DESIGNER_ID`

Reply:
Removed DESIGNER_ID

Reply:
Reolved the conflict